scripts/scil_tractogram_pairwise_comparison.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+This script is designed to compare and help visualize differences between
+two tractograms. Which can be especially useful in studies where multiple
+tractograms from different algorithms or parameters need to be compared.
+
+The difference is computed in terms of
+- A voxel-wise spatial distance between streamlines, out_diff.nii.gz
+- A correlation (ACC) between streamline orientation (TODI), out_acc.nii.gz
+- A correlation between streamline density, out_corr.nii.gz
+"""
+
+import argparse
+import logging
+import os
+
+import nibabel as nib
+
+from scilpy.io.utils import (add_overwrite_arg, add_reference_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist,
+                             assert_output_dirs_exist_and_empty,
+                             add_processes_arg,
+                             add_verbose_arg,
+                             is_header_compatible_multiple_files,
+                             load_tractogram_with_reference,
+                             validate_nbr_processes)
+from scilpy.tractograms.tractogram_operations import tractogram_pairwise_comparison
```

Comment:
Also it is important to note that from big script it is possible that when we extract the main() and put it somewhere, it fits in two places (some script deal with maps, labels and tractogram and it can be difficult to pick the best one)

If you create scilpy.tractograms.reproducibility_measures
Do you create scilpy.bundles.reproducibility_measures ?

Reply:
I like scilpy.tractanalysis.reproducibility_measures it seems to fit our needs here. We'll see in the future if we need to create scilpy.tractograms.reproducibility_measures or scilpy.bundles.reproducibility_measures